scripts/functional-tests.sh

```diff
@@ -582,6 +582,34 @@ test_ilab_chat_server_logs(){
     fi
 }
 
+test_ilab_help_without_config() {
+    # Test the help without config file, and it will show help message
+    result=$(ilab --config=test-test.yaml model evaluate --help)
+    if echo "$result" | grep -F 'Usage: ilab model evaluate [OPTIONS]'; then
+        echo "Test passed: ilab model evaluate help message found."
+    else
+        echo "Error: ilab model evaluate help message not found."
+        exit 1
+    fi
+
+    result=$(ilab --config=test-test.yaml config show --help)
```

Comment:
Should we be passing the config file for these tests if our goal is testing that `--help` will print when it doesn't have a config?

Reply:
I think is need, just make sure it go to `else` for testing..
```
 if config_file == "DEFAULT":

elif os.path.isfile(config_file):
 
else:
```

Reply:
Ah I see, you're passing in a config file that doesn't exist. That makes sense.